bin/s2nd.c

```diff
@@ -135,6 +135,13 @@ static char dhparams[] =
     "HI5CnYmkAwJ6+FSWGaZQDi8bgerFk9RWwwIBAg==\n"
     "-----END DH PARAMETERS-----\n";
 
+const unsigned char tick_key_name[16] = "2016.07.26.15\0";
```

Comment:
why tick_?  why not ticket_?

also, should you add the option of taking the key as a command line argument (literal value or file?) or generate a random key and print to console/write to file?

Reply:
Will add an option to take key file as a command line argument.